synapseAnnotations/data/experimentalData.json

```diff
@@ -750,6 +755,11 @@
         "description": "",
         "source": ""
       },
+      {
+        "value": "embryonic",
```

Comment:
this probably merits further discussion - is this a tissue? or a feature of a tissue? for example, what about embryonic blood? or embryonic brain?

Reply:
Agree, we should definitely discuss

Reply:
I added a related issue #405 to our next milestone! :)

Reply:
Agreed in 2018-08-28 meeting that this should be renamed to "embryonic tissue"

Reply:
Updated to embryonic tissue.